metaflow/metaflow_config.py

```diff
@@ -26,6 +26,7 @@
 DEFAULT_MONITOR = from_conf("DEFAULT_MONITOR", "nullSidecarMonitor")
 DEFAULT_PACKAGE_SUFFIXES = from_conf("DEFAULT_PACKAGE_SUFFIXES", ".py,.R,.RDS")
 DEFAULT_AWS_CLIENT_PROVIDER = from_conf("DEFAULT_AWS_CLIENT_PROVIDER", "boto3")
+DEFAULT_AZURE_CLIENT_PROVIDER = from_conf("DEFAULT_AWS_CLIENT_PROVIDER", "azure-default")
```

Comment:
Should this be `"DEFAULT_AZURE_CLIENT_PROVIDER"`

Reply:
It should!